routes/target.js

```diff
@@ -0,0 +1,37 @@
+var express = require("express");
+var router = express.Router();
+const db = require("../model/helper");
+
+// Postman Test = OK http://localhost:5000/api/target/test
+router.get("/test", (req, res) => {
+    res.send("Test route is working!");
+  });
+
+// Postman Test = OK http://localhost:5000/api/target/targets
+router.get("/targets", async (req, res) => {
+    console.log('GET /api/target/targets route hit');
+    try {
+      // Search the database to retrieve all the targets
+      const result = await db("SELECT * FROM target");
```

Comment:
`targets`

Reply:
The table name which we are querying is "target".